frontend/src/pages/about/PageOutreach.vue

```diff
@@ -0,0 +1,146 @@
+          </p>
+          <ul>
+            <li>
+              The Mondo Outreach calls aim to bring together stakeholders and
+              clinical terminology experts to discuss specific use cases and
+              requirements. Our goal is to improve Mondo with a focus on our
+              user needs.
+            </li>
+            <li>
+              These calls are scheduled for the second Thursday of every month.
+              You can find more information about the calls
+              <a
+                href="https://mondo.monarchinitiative.org/pages/workshop/#outreach"
+                target="_blank"
+                >here</a
+              >.
+            </li>
+            <li>
+              You can find the schedule with Zoom information
+              <a
+                href="https://docs.google.com/document/d/1MStpDyh9LOZYJTjLtnpOsNYc3HaeU-bz0CHAI9xOjfQ/edit#heading=h.6sqkx1xhumse"
+                target="_blank"
+                >here</a
+              >.
+            </li>
+          </ul>
+        </li>
+        <li>
+          <a href="https://linkml.io/linkml/" target="_blank"
+            >LinkML (the Linked data Modeling Language)</a
+          >
+          <p>
+            LinkML is a flexible modeling language that allows you to author
+            schemas in YAML that describe the structure of your data.
+          </p>
+          <p>
+            LinkML is used to model the Monarch data model, and we welcome your
+            contributions to the LinkML community. You can learn more
+            <a
+              href="https://linkml.io/linkml/faq/contributing.html"
+              target="_blank"
+              >here</a
+            >.
+          </p>
+        </li>
+      </ul>
+    </AppSection>
+    <AppSection>
+      <AppHeading>Follow us to learn more!</AppHeading>
+      <p>
+        Stay updated on the latest from the Monarch Initiative by joining our
+        <a
+          href="https://groups.google.com/a/tislab.org/g/monarch-users"
+          target="_blank"
+          >user mailing list</a
+        >.
+      </p>
+      <AppFlex direction="col" align-h="center">
+        <!-- <AppTile
+          to=""
+          icon=""
+          title=""
+          subtitle=""
+          design="small"
+        /> -->
+        <AppButton
+          text="Monarch GitHub Organization"
+          to="https://github.com/monarch-initiative"
+        />
+        <AppButton
+          text="Monarch LinkedIn Profile"
+          to="https://www.linkedin.com/company/the-monarch-initiative/"
+        />
+        <AppButton text="Monarch Blog" to="https://monarchinit.medium.com/" />
+        <AppButton
+          text="Monarch YouTube Channel"
+          to="https://www.youtube.com/@monarchinitiative"
+        />
+      </AppFlex>
+    </AppSection>
+    <AppSection>
+      <AppHeading>For questions or technical support</AppHeading>
+      <p>
+        We welcome all questions, requests, and feedback! Please submit your
```

Comment:
Is this section necessary? Perhaps we could move some of this text to the Help page (where I'd argue it belongs), and then here just have a big button link to it.

Reply:
yes. agree.